runtime/src/main/scala/com/ing/baker/runtime/akka/actor/serialization/protomappings/ProcessStateMapping.scala

```diff
@@ -32,5 +32,5 @@ class ProcessStateMapping extends ProtoMap[ProcessState, proto.ProcessState] {
             value <- ctxFromProto(protoValue)
           } yield (name, value)
         }
-      } yield ProcessState(processId, ingredients.toMap, message.eventNames.toList)
+      } yield ProcessState(processId, ingredients.toMap, message.events.map(ctxFromProto(_).get))
```

Comment:
we should not do `.get` here, we need to do `message.events.toList.traverse(ctxFromproto(_))` and add that to the `for` comprehension